rulesetMD.xml

```diff
@@ -7,36 +7,6 @@
 >
     <description>ShopSys Coding Standards</description>
 
-    <!--
-        <rule ref="rulesets/cleancode.xml/ElseExpression" />
```

Comment:
The purpose of this is to document why we do not use the rule. How else would you recommend to overcome the situation that somebody will spend unnecessary time trying to enable the rule and figuring out that it cannot be used and disabling it again (over and over)?

Reply:
I guess this is already covered in #7 

Also these *WIP code* should be documented in issues, otherwise it's only dead code and that should never be part of `master` branch of open-source project.

Reply:
I am not sure if you understand. If you are only one maintainer you probably remember which rules you enabled and which not and why.

But somebody else might see this ruleset and say "hey, why don't you use *ElseExpression* rule"? And I will look at it and say "hmm, seems like a good idea, lets enable it". Then I will enable it and see why we did not enable it and say to myself "oh, that's why". This already happened. I did not keep the rules commented for no reason.

Also, this is not a *dead code*. It's in-line documentation of reason why we do not enable the rule. And it's not a *WIP code* too because we actually do not plan to rewrite the code in the future to adhere these rules.

Reply:
We could document the reasons for not enabling it in docs as well but I think that the ruleset file it the best place for it because this is the only place where you are sure that somebody will look when trying to enable some rule (people do not expect to find such information in documentation).

Well, maybe when this repository is all about coding standards then *docs* directory is a good place for this (but in case of a ruleset file in a normal project I would definitely put the info directly in the ruleset file).

Reply:
I understand you've been discussion this internally and there is more work then meets the eye.


I think **end user** like me will ask **3 important questions**:

- 1. **Do you use PSR-2 set?**
- 2. **What extra do you offer?**
- 3. Both conclude to: **Why should I use this package over clean PHP_CodeSniffer with own ruleset?**

Motivation of chosen rules or not-chosen rules is not so relevant for majority of users. I'll mention it again, I didn't noticed this file until I send PR. That should not happen. User should read everything important about this right from the `README.md`.

I'd invest into answering these 3 question rather then to describing known rules.